style.css

```diff
@@ -1,3 +1,97 @@
 body {
-  background: yellow;
+  margin: 0;
+  font-family: sans-serif;
+  font-size: 1rem;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.navbar {
+  display: block;
+  background: #2e84f6 ;
+  overflow:auto;
+  color: white;
+  font-size: .5rem;
+  font-family: "Source Sans Pro", sans-serif;
+
+}
+
+.navbar_title {
+  float: left;
+  padding-left: 1rem;
+
+}
+.navbar_portfolio {
+  float: right;
+  padding-right: 1rem;
+}
+
+.navbar_blog {
+  float: right;
+  padding-right: 1rem;
+}
+
+.navbar_contact {
+  float: right;
+  padding-right: 1rem;
+}
+
+.container {
+  max-width: 50%;
+    margin: 0 auto;
+    padding-top: 2rem;
+    margin: 1rem;
+    width: calc(100% / 2 - 1rem - 1rem);
+    float: left;
+    padding-right: 4rem;
+  }
+
+.profile_user {
+  overflow: auto;
+  margin-right: -3rem;
+  margin-left: -2.5rem;
+  background: #ffe32b;
+}
+
+.blog {
+  box-shadow: 0.2rem 0.2rem 0.4rem rgba(0, 0, 0, 0.4);
+  color: white;
+  display: inline-block;
+
+  padding-right: 3rem;
+  margin-left: 25%;
+ }
+
+.image {
+      border-radius: 50%;
+      position: absolute;
+      transform: translate(-50%, -50%);
+      margin-top: 4rem;
+      box-shadow: 0.2rem 0.2rem 0.4rem rgba(0, 0, 0, 0.4);
+}
+
+.week {
+      color: white;
+       background: #2e84f6;
+       position: absolute;
```

Comment:
Since `week` is position absolute, remember to make the element that it should be compared to when talking about left, right, top, bottom and make that `position: relative`. In this case that probably is the `blog` or `profile_user` div.

Then remove all margins and use `left` to center this compared to the yellow area.